libgraph/include/katana/GraphTopology.h

```diff
@@ -140,12 +140,13 @@ class KATANA_EXPORT GraphTopology : public GraphTopologyTypes {
     return static_cast<Node>(d);
   }
 
-  nodes_range nodes(Node begin, Node end) const noexcept {
-    return MakeStandardRange<node_iterator>(begin, end);
-  }
+  ///@param node node to get degree for
```

Comment:
nit: space between `///` and text

nit: I remember @arthurp suggesting `\p node` instead of `@param node`; similarly `\returns ...`. Don't remember why.

Reply:
In this case @param is correct. The case where it's wrong is in the middle of text. The way to think about it is that `@param` "defines" a parameters and `@p` refers to it. 

And `@` vs `\` doesn't matter. They are the same to doxygen.

Reply:
Thanks! I'll remember it going forward (hopefully).